docs/compose/AppSearchModal/README.md

```diff
@@ -40,13 +46,15 @@ BpkAppSearchModal(
     inputText = destination.value,
     inputHint = stringResource(id = R.string.text_field_hint),
     results = BpkAppSearchModalResult.Content(
-        sections = listOf(BpkSection(...)),
-        shortcuts = listOf(BpkShortcut(...)),
+        sections = listOf(BpkSection(...)
```

Comment:
looks like the formatting broke in the readme - can you please fix that? it's quite hard to parse right now

Reply:
this still looks funky - i think the auto formatter gets confused about the `...` - sections + shortcuts are at the same level, but it looks like shortcuts is a level up because of the formatting - have a look at the diff here to make it clearer what i mean